docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -393,12 +423,19 @@ Start by adding the following dependencies to your test project:
 testImplementation("io.rest-assured:rest-assured")
 testImplementation("io.quarkus:quarkus-junit5")
 ----
+====
+
+//@sberyozkin - It might be good to add a stament here about the different mechanisms or methods to integration testing. For example, WireMock... etc
+
 
 [[integration-testing-wiremock]]
-==== Wiremock
+==== WireMock
 
-Add the following dependencies to your test project:
+You can also use link:https://wiremock.org/[WireMock] for integration testing.
 
+. Add the following dependencies to your test project:
++
+====
```

Comment:
This noise makes the build of downstream docs to be stuck

there should be no `+` and `====` on solo lines

CC @sberyozkin @michelle-purcell

Reply:
@rsvoboda @michelle-purcell It took me may be 5 rounds to get past the downstream task, I've ended removing all of standalone `+` and `====` and it looked like it fixed. Michelle, not sure if te end result would be what you expected it to be without these characters, so I guess you should have a look before me merging this PR

Thanks